src/__test__/bitDepth1.read.test.ts

```diff
@@ -23,6 +23,9 @@ const data = {
   bitDepth: 1,
   components: 1,
   channels: 1,
+  logicalColorSpace: 2,
+  compression: 0,
+  colorMasks: [16711680, 65280, 255],
```

Comment:
Could you write those numbers with their hexadecimal notation?

(there are other occurences than this one)